pages/account/manage/event/[[...data]].js

```diff
@@ -105,6 +106,10 @@ export default function ManageEvent({ BASE_URL, event }) {
   const handleSubmit = async (e) => {
     e.preventDefault();
 
+    if (document.activeElement === tagInputRef.current) {
+      console.log('testing');
```

Comment:
Please remove debug code

Reply:
Sorry I forgot to remove it before now I've removed it and pushed the changes. Thanks for addressing this.